src/SDK/CGlobalVars.h

```diff
@@ -18,7 +18,4 @@ class CGlobalVars
 	void* pSaveData;
 	bool m_bClient;
 	bool m_bRemoteClient;
-private:
-	int nTimestampNetworkingBase;
-	int nTimestampRandomizeWindow;
```

Comment:
this is a part of csgo hooks, you probably shouldn't delete that

Reply:
Clang said that this variables is unused.

Reply:
It may be unused in cheat but those are present in memory created by game.

Reply:
I understand, but it nothing broken...

Reply:
Yes, It won't break anything, because they are at the end. Means no addresses are being broken.
Think about it that way:
This is a Header:
Variable 2 Bytes
Variable 2 Bytes
Variable 4 Bytes
Variable 8 Bytes

In this case deleting the last variable if unused won't change anything, because if read the first 3 are still interpreted as the type they are supposed to.
But:
Variable 2 Bytes
Variable 2 Bytes
Variable 8 Bytes

This would break it. Because suddenly a 4 Byte Variable is interpreted as a 8 Byte Variable and the variable behind the 4 Byte are interpreted as the "second part" of the 8 Byte Variable.

Developers like to keep those lines because they might be needed later and in general they dont like to change the games internal structs/objects etc...

Reply:
yeah even if this is unused, it's still a part of the layout. We are using typed pointers into the game